scripts/lib/provision_vm.py

```diff
@@ -11,7 +11,7 @@
 
 PY2 = sys.version_info[0] == 2
 
-ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
this should probably move to tools/lib, not scripts/lib.  The latter is for things used in prod as well.